blog/2023/12-11-roku-v200/index.mdx

```diff
@@ -0,0 +1,121 @@
+---
+title: Roku Version 2.0.0
+description: I Want a Rokupotamus for Christmas
+authors: 1hitsong
+date: 2023-12-11
+slug: roku-200
+tags: [release, roku]
+---
+
+When we say this 2.0.0 release is a major milestone, we honestly mean it. New features and bug fixes abound, but even behind the scenes the updates continue all the way to the foundation with us updating every single file in the app to support a new programming language. You can't get more monumental than that!
+
+Read more to learn about some of the exciting new features.
+
+\- 1hitsong
+
+{/* truncate */}
+
+Requires minimum server version of 10.8.1
+
+### 👥 User Select Screen / Auto Login
+
+![Screenshot of the user select screen in the Jellyfin Roku app](./UserSelect.gif)
+
+When launching the Jellyfin app, it now presents you with the list of all public users and any other remembered users for you to login as.
```

Comment:
This is only true for new installs because of the registry migration. Current installs will have the feature disabled

Reply:
Added this text: **Note:** Users upgrading from 1.6.6 will have this global setting enabled by default and will automatically login as the last user logged in to the 1.6.6 app.